src/chapters/data-and-variables/other-operators.rst

```diff
@@ -0,0 +1,100 @@
+Other Operators
+===============
+
+The String Operator ``+``
+--------------------------
+
+.. index::
+   pair: string; concatenation
+
+So far we have only seen operators that work on operands which are of type ``number``, but there are operators that work on other data types as well. In particular, the ``+`` operator can be used with ``string`` operands to **concatenate**, or join together two strings.
+
+.. admonition:: Example
+
+   ``"Launch" + "Code"`` evaluates to ``"LaunchCode"``
+
+Let's compare ``+`` used with numbers to ``+`` used with strings.
+
+.. sourcecode:: js
+   :linenos:
+
+   console.log(1 + 1);
+   console.log("1" + "1");
+
+**Output**
+
+::
+
+   2
+   11
+
+..pull-quote:: The operator ``+`` behaves differently based on the data type of its operands.
```

Comment:
Space between .. and pull-quote

Reply:
This also seemed like a poor use of pull-quote, so I removed it completed in favor of bolding the important point.